src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala

```diff
@@ -204,6 +224,15 @@ class LibraryApiServiceSpec extends BaseServiceSpec with LibraryApiService {
           }
         }
       }
+      "GET on " + libraryGroupsPath - {
+        "should return the all broad users group" in {
+          new RequestBuilder(HttpMethods.GET)(libraryGroupsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(libraryRoutes) ~> check {
+            status should equal(OK)
+            val respdata = response.entity.asString.parseJson.convertTo[Seq[String]]
+            assert(respdata.forall { g => FireCloudConfig.ElasticSearch.discoverGroupNames.contains(g) })
```

Comment:
The `forall` was my suggestion. I think what you need is to check that the set of `FireCloudConfig.ElasticSearch.discoverGroupNames` contains all of the elements in `respdata`. Simple set equality will fail as soon as we have more than one discoverable group and the user is only adding one of them to the workspace.